### PR TITLE
Fixed #31505 -- Doc'd possible email addresses enumeration in PasswordResetView.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

File: docs/topics/auth/default.txt [hunk 1]
```diff
@@ -1238,6 +1238,14 @@ implementation details see :ref:`using-the-views`.
     :class:`~django.contrib.auth.forms.PasswordResetForm` and use the
     ``form_class`` attribute.
 
+    .. note::
+
+        Be aware that sending an email cost extra time, hence you may be
```
Comment (carltongibson) [inline docs/topics/auth/default.txt:1243]: s/cost/costs/

Comment (felixxm) [reply docs/topics/auth/default.txt:1243]: Thanks, fixed.

**Comment by apollo13** [comment]
The text looks good, though I am wondering if it makes sense to add something like this at all. My next question as someone reading those docs would be "so how do I actually fix it"?

**Comment by felixxm** [comment]
@apollo13 Thanks for the review :+1: I think it's still worth to call out this potential email addresses leak.

**Pull request merged**